scripts/drop-oracle-tables.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export DYLD_LIBRARY_PATH=/opt/oracle/instantclient/
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+/opt/oracle/instantclient/sqlplus system/oracle@192.168.99.100:49161/xe < "$DIR/drop-tables.sql"
```

Comment:
I agree this is bad practice, but we already have hardcoded IPs in the knexfile. We should probably change all of them in one go.